.changeset/thirty-hairs-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/rss': patch
+---
+
+This commit addresses a quality-of-life concern when setting up a RSS feed using collections. Specifically, it provides more context to the error message thrown when the object passed to the `items` property is missing any of the three required keys or if one of those keys is mistyped.
```

Comment:
```suggestion
Improves the `@astrojs/rss` error message thrown when the object passed to the `items` property is missing any of the three required keys or if one of those keys is mistyped.
```
Appreciate the precision of the original statement! Just formatted according to our normal structure (starting with the verb) so the reader will hit the actual PR contributions right away.

Reply:
@sarah11918 Happy to take the suggestion! Simplifies it to exactly what's necessary.